lib/commands/connect/auth_connection.js

```diff
@@ -38,10 +38,10 @@ module.exports = {
               if (context.flags.login) {
                 args.login_url = context.flags.login;
               }
-              api.request(context.auth.password, 
+              api.request(context.auth.password,
                 'POST', '/api/v3/connections/' + connections[0].id + '/authorize_url', args).then(function(response) {
                   let redir = response.json.redirect;
-                  console.log("Auth response ", redir);
+                  console.log("Redirecting to Salesforce for authorization");
```

Comment:
I removed it intentionally, because it's a pretty massive wall of a URL. But yeah, I suppose there might be cases where it'll need to be copied and pasted manually.